src/org/rascalmpl/courses/Recipes/Common/WordCount/CountInLine2/CountInLine2.concept

```diff
@@ -21,11 +21,11 @@ include::{LibDir}demo/common/WordCount/CountInLine2.rsc[tags=module]
 ----
 
                 
-The pattern `/^\W*<word:\w+><rest:.*$>/` can be understood as follows:
+The pattern `/^\W*\w+<rest:.*$>/` can be understood as follows:
 
 *  The `^` makes it _anchored_, only matches at the begin of the substring `S`.
 *  `\W*` matches zero or more non-word characters.
-*  `<word:\w+>` matches one or more word characters and assigns the result to the variable `word`.
```

Comment:
If removing the explanation about assigning names to a group from this case, we need to discuss it for the next line